app.server/data/catalog/catalog.ts

```diff
@@ -0,0 +1,157 @@
+import * as config from '../../config/config';
+import { GetCollectionsResult, WMSData } from '../../handlers/collections/models';
+import { Query } from '../../../app.client/components/models/Query';
+import { GetProductsResult } from '../../handlers/products/models';
+import axios from 'axios';
+
+import urljoin = require('url-join');
+import turf = require('turf');
+
+export class Catalog {
+  collectionSearchEndpoint: string = 'search/collection';
+  collectionSearchOGCPattern: string = 'scotland-gov/lidar/ogc';
+  collectionSearchLidarPattern: string = 'scotland-gov/lidar/phase*';
+  productSearchEndpoint: string = 'search/product';
+
+  geoserverLayers: { [id: string]: { base_url: string, name: string } };
+
+  private addToGeoserverLayers(name: string, wms: WMSData) {
+    if (this.geoserverLayers) {
+      this.geoserverLayers[name] = wms;
+    } else {
+      this.geoserverLayers = {
+        [name]: wms
+      };
+    }
+  }
+
+  async getOGCServiceList(): Promise<{ [id: string]: WMSData }> {
+    let maxBbox = [-180.0, -85.06, 180.0, 85.06]
```

Comment:
What map box is this?

Reply:
World coverage, was supposed to be the limits of the current data, but there is no harm in just fetching everything that could exist here

Reply:
Ah great, well let's make the identifier explain what it is then? Perhaps:

    let entireWorldBbox = [-180.0, -85.06, 180.0, 85.06]

Otherwise it's a bit of a magic string (well, array!)

Reply:
made change